src/components/SearchFilters/SearchFilters.js

```diff
@@ -0,0 +1,103 @@
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+import omit from 'lodash/omit';
+
+import {
+  AcqDateRangeFilter,
+} from '@folio/stripes-acq-components';
+
+import { MultiSelectionFacet } from '../MultiSelectionFacet';
+import { useSectionToggle } from '../../hooks';
+import { useFacets } from '../../queries';
+
+const FACETS = {
+  HEADING_TYPE: 'headingType',
```

Comment:
We already have constant for it in `searchResultListColumns`, why you don't use it?

Reply:
Because I think these are different entities. In `searchResultListColumns` we have columns for results table, but this constant will contain facet filters. We have a couple more stories for `Created by` and `Updated by` facets.